debian/pantos-service-node.sysuser

```diff
@@ -0,0 +1 @@
+pantos-service-node  defaults
```

Comment:
Missing new line at the end.

Reply:
We shouldn't add empty lines on this file